src/Microsoft.AspNet.Http/HttpContextFactory.cs

```diff
@@ -9,6 +9,10 @@ public class HttpContextFactory : IHttpContextFactory
     {
         private IHttpContextAccessor _httpContextAccessor;
 
+        public HttpContextFactory() : this(null)
```

Comment:
@halter73 Wasn't the default DI container really bad at choosing between multiple constructors?